Rx-VSCDesign-light/theme.css

```
@@ -26,74 +26,91 @@ hexview_t {
 
 QToolBar {
   background-color: #efefef;
-  border          : none;
-  spacing         : 5px;
-  border          : 1px solid #eeeeee;
+  border: none;
+  spacing: 0.2em;
+  border: 0.5px solid #dddddd;
+  padding: 0.5em 0.1em;
 }
 
 QToolBar:active {
-  background-color: #dfdfdf;
+  background-color: #f0f0f0;
 }
 
-QToolBar QWidget {
+
+QToolButton {
+  min-width: 1.2em;
+  height: 1.2em;
+  border: 0px solid black;
   background: transparent;
-  border    : 1px solid #eeeeee;
+}
+
+QToolButton:hover {
+  background: #20ffffff;
```

```suggestion
  background: #20000000;
```

button highlight in light theme should not be `#ffffff` or alpha values.